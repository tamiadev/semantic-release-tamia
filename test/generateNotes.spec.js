const generateNotes = require('../generateNotes');

test('Generate list of fixes', async () => {
	const commits = [
		{
			subject: 'Fix: Very cool commit',
		},
		{
			subject: 'Fix(zzz): Another cool commit',
		},
	];
	const result = await generateNotes({}, { commits });
	expect(result).toMatchSnapshot();
});

test('Return commit body for change log commits', async () => {
	const commits = [
		{
			subject: 'Changelog:',
			body: 'My amazing changelog',
		},
	];
	const result = await generateNotes({}, { commits });
	expect(result).toMatchSnapshot();
});
