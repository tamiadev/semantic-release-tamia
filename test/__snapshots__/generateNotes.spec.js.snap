// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Generate list of fixes 1`] = `
"* **Fixed:** Very cool commit
* **Fixed:**: Another cool commit"
`;

exports[`Return commit body for change log commits 1`] = `"My amazing changelog"`;
